kratos.gid/apps/Common/xml/Solvers.xml

```diff
@@ -39,7 +39,7 @@
         </inputs>
     </solver>
 
-    <solver n="AMGCL" pn="AMGCL" SolverType="Standard" Parallelism="OpenMP">

```

Comment:
Guys, I've just realized that if we rename the `AMGCL` to `amgcl`the GUI takes the Trilinos AMGCL since its name is also `amgcl`. If the serial settings coincide with the MPI ones we can think on having a unique `amgcl`entrancy. Blocking until we ensure that.

Reply:
As far as I understand the settings are exactly the same since the MPI version is derived from the serial one. Having this in mind, we need to decide which AMGCL features do we want to expose to the user.

Reply:
As far as I understand the settings are exactly the same since the MPI version is derived from the serial one. Having this in mind, we need to decide which AMGCL features do we want to expose to the user.